### PR TITLE
editorial changes to headlines to make the TOC better readable

### PR DESCRIPTION
**Pull request by tlossen**
so that the table of contents can serve as a compact representation of the guidelines.

### CODE REVIEW

File: common-data-objects/CommonDataObjects.md [hunk 1]
```diff
@@ -116,7 +116,7 @@ responses:
         
 ```
 
-## {{ book.must }} An error message must not contain the stack trace.
+## {{ book.must }} Do not expose Stack Traces.
```
Comment (whiskeysierra) [inline common-data-objects/CommonDataObjects.md]: I'd drop the period at the end

File: events/events.md [hunk 2]
```diff
@@ -8,21 +8,21 @@ Nakadi is a key element for data integration in the architecture, and the standa
 
 Events are part of a service’s interface to the outside world equivalent in standing to a service’s REST API. Services publishing data for integration must treat their events as a first class design concern, just as they would an API. For example this means approaching events with the "API first" principle in mind [as described in the Introduction](../Introduction.md) and making them available for review.
 
-## {{ book.must }} Events should define useful business resources
+## {{ book.must }} Events define useful business resources
```
Comment (whiskeysierra) [inline events/events.md]: *must define*?

Comment (whiskeysierra) [inline events/events.md]: It's twice then, because it already has the *Must* prefix.

File: naming/Naming.md [hunk 3]
```diff
@@ -36,11 +36,11 @@ See also: [HTTP Headers are case-insensitive
 Use [this list](http://en.wikipedia.org/wiki/List_of_HTTP_header_fields) and mention its support in
 your OpenAPI definition.
 
-## {{ book.must }} Always Pluralize Resource Names
+## {{ book.must }} Pluralize Resource Names
 
 Usually, a collection of resource instances is provided (at least API should be ready here). The special case of a resource singleton is a collection with cardinality 1.
 
-## {{ book.could }} First Path segment May be /api
+## {{ book.could }} Use /api as first Path Segment
```
Comment (whiskeysierra) [inline naming/Naming.md]: Side note, I'd like to drop this rule.

**Comment by whiskeysierra** [comment]
:+1:

File: common-data-objects/CommonDataObjects.md (hunk 1 above)
Comment (tlossen) [inline common-data-objects/CommonDataObjects.md]: done

File: common-data-objects/CommonDataObjects.md [hunk 4]
```diff
@@ -116,7 +116,7 @@ responses:
         
 ```
 
-## {{ book.must }} An error message must not contain the stack trace.
+## {{ book.must }} Do not expose Stack Traces
```
Comment (whiskeysierra) [inline common-data-objects/CommonDataObjects.md]: Now we have two different kinds of wording for headlines:

1. *Do not expose Stack Traces*, imperative and works without the **must**
2. *Events define useful business resources*, just a statement without the **must**

Should we be consistent here?

Comment (tlossen) [inline common-data-objects/CommonDataObjects.md]: ideally, i would like them all to be all in imperative form -- but for some headlines it doesn't work well.

**Comment by tlossen** [comment]
ok, headlines are in imperative form now.

**Comment by tlossen** [comment]
("Ensure to Use the Force, Luke!")

**Comment by whiskeysierra** [comment]
:+1:

**Comment by tfrauenstein** [comment]
👍

**Pull request merged**